appengine-plugins-core/src/main/resources/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -785,39 +785,6 @@
       }
     ]
   },
-{
-    "name": "Phishing Protection",
-    "id": "phishingprotection",
-    "serviceName": "phishingprotection.googleapis.com",
-    "serviceRoles": [
-      "roles/cloudiot.admin",
-      "roles/cloudiot.deviceController",
-      "roles/cloudiot.editor",
-      "roles/cloudiot.provisioner",
-      "roles/cloudiot.viewer",
-      "roles/cloudiot.serviceAgent"
-    ],
-    "documentation": "https://cloud.google.com/phishing-protection/docs/apis",
-    "description": "A phishing countermeasure that helps to detect phishing attacks.",
-    "transports": ["grpc"],
-    "clients": [
-      {
-      "name": "Java Client Libraries for Phishing Protection",
```

Comment:
Nice investigation

Reply:
I get the same error in master-20231129 branch too.

Reply:
@JoeWang1127 Sent an email, CC-ed you.

Reply:
Not sure why an unit test has real network connection.